client/src/redux/reducers.js

```diff
@@ -0,0 +1,27 @@
+import { combineReducers } from 'redux';
```

Comment:
Reducers cause the actual mutations to the store. All reducers will `listen` for all actions, but only mutate for the action it is concerned with.